eng/Version.Details.xml

```diff
@@ -1,48 +1,52 @@
+      <Sha>f263e24ff720381ba0c17f10ce0b4d1725e1c062</Sha>
     </Dependency>
-    <Dependency Name="System.Diagnostics.DiagnosticSource" Version="4.6.0-preview4.19122.4">
+    <Dependency Name="System.Diagnostics.DiagnosticSource" Version="4.6.0-preview4.19123.2">
+      <CoherentParentDependency>Microsoft.NETCore.App</CoherentParentDependency>
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>cbf462058cf0aa96c171f84604e575be9f41830a</Sha>
+      <Sha>327def063eb5eddb723edcee84815d9495b4021c</Sha>
     </Dependency>
-    <Dependency Name="System.Text.Encodings.Web" Version="4.6.0-preview4.19122.4">
+    <Dependency Name="System.Text.Encodings.Web" Version="4.6.0-preview4.19123.2">
+      <CoherentParentDependency>Microsoft.NETCore.App</CoherentParentDependency>
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>cbf462058cf0aa96c171f84604e575be9f41830a</Sha>
+      <Sha>327def063eb5eddb723edcee84815d9495b4021c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.0.0-preview4-27422-5">
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.0.0-preview4-27424-1">
+      <CoherentParentDependency>Microsoft.NETCore.App</CoherentParentDependency>
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>b344ca9aafbc18e3a75cd5237ff9c1b30aa15987</Sha>
+      <Sha>e1e36e48fb64a7637cbb3d340e44fb97256dfb3c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview4-27422-5">
+    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview4-27424-1">
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>b344ca9aafbc18e3a75cd5237ff9c1b30aa15987</Sha>
+      <Sha>e1e36e48fb64a7637cbb3d340e44fb97256dfb3c</Sha>
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
     <!-- Listed as a dependency to workaround https://github.com/dotnet/cli/issues/10528 -->
-    <Dependency Name="Microsoft.NETCore.Platforms" Version="3.0.0-preview4.19122.4">
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="3.0.0-preview4.19123.2">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>cbf462058cf0aa96c171f84604e575be9f41830a</Sha>
+      <Sha>327def063eb5eddb723edcee84815d9495b4021c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.BenchmarkRunner.Sources" Version="3.0.0-preview4.19122.10">
+    <Dependency Name="Microsoft.AspNetCore.BenchmarkRunner.Sources" Version="3.0.0-preview4.19122.13">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>748155b796cb10505366cb1fcae54e6ba3ee9567</Sha>
+      <Sha>f263e24ff720381ba0c17f10ce0b4d1725e1c062</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Testing" Version="3.0.0-preview4.19122.10">
+    <Dependency Name="Microsoft.AspNetCore.Testing" Version="3.0.0-preview4.19122.13">
+      <CoherentParentDependency>Microsoft.NETCore.App</CoherentParentDependency>
```

Comment:
Why has this been added only for this dependency and not all dependencies from aspnet/Extensions?

Reply:
I didn't bother to update the `... .Sources` dependencies.